### DISCUSSION TITLE
[suggestion]: Kick suspicious bucket placers

### DISCUSSION THREAD

**Discussion by ghost** [GriefPrevention/GriefPrevention#123] [Ideas]
I love this plugin, worked nice and smooth, no problems with it so far

This is not an issue but a suggestion,
why not add an option to warn then kick the player after placing too many suspicious buckets

I've programed this with Bash trough Linux by reading the log and sending a command trough screen of the minecraft server but it would be nice to have it available trough the config file

I mean if you're already writing it to the log why not count it and take action against it
let's say 10 suspicious placings = warning , 15 = kick 

I'm trying to automatically protect the server from harm so this would be a huge plus for someone who can't program

**Comment by RoboMWM** [comment]
I've actually turned the logging for this off on my server because my players use buckets for all their various contraptions, which happen to be above ground, lol.

**Comment by ghost** [comment]
Well i need it for my bash script, you see it's not a problem that they use buckets for their stuff but i have a few creative worlds on my server and what they do is place random lava buckets everywhere so if they put 10-20 buckets your plugin finds suspicious i warn them to stop it and if they don't stop i kick them out.
What i do is just tail your log and count how many times it logged the problem for each player

Players don't like the idea of having to claim land to build in creative worlds so i set the claims mode to survival to keep them from complaining but now everyone can grief freely,
I'm not really sure why would anyone waste so much time just to fill the flat world with lava

I always have a backup i can restore but why not prevent if from happening in the first place

--Cheers

**Comment by RoboMWM** [comment]
> Players don't like the idea of having to claim land to build in creative worlds so i set the claims mode to survival to keep them from complaining but now everyone can grief freely,

Could always use plots, lol. But yea - creative mode claims automatically restrict liquids from flowing out of claims, and cleans them up when they're removed.
